apiserver/facades/client/client/client.go

```diff
@@ -113,6 +118,19 @@ func (c *Client) checkIsAdmin() error {
 
 // NewFacade provides the required signature for facade registration.
 func NewFacade(ctx facade.Context) (*Client, error) {
+	return newFacade(ctx)
+}
+
+// NewFacadeV1 provides the required signature for facade registration.
```

Comment:
Can you change this comment (and the one for NewFacade above) to something like "NewFacadeV1 creates a version 1 Client facade to handle API requests."? The comment that's here (I know you didn't write it) is too focused on the specific change that was being made when they were converting facades to have a new signature - it doesn't actually say what the function does.